tests/deep/Tools/Radiation/Models/Cone/Bremsstrahlung/Test_BremsstrahlungScreenedEmission.cpp

```diff
@@ -0,0 +1,221 @@
+/**
+ * Test of the bremsstrahlung screned emission implemented in the cone model.
+ * The following tests are included:
+ *
+ *   [ ] - Emission formulas.
+ *
+ */
+
+#include "Tools/Radiation/RadiationParticle.h"
+#include "Tools/Radiation/Models/Cone/ConeBremsstrahlungScreenedEmission.h"
+#include "Test_BremsstrahlungScreenedEmission.h"
+
+using namespace std;
+using namespace __Radiation;
+
+const unsigned int
```

Comment:
Add a comment above this line regarding what these constants mean

Reply:
...actually, with my other suggestions implemented, these should just be removed.